src/WingetCreateCLI/Commands/CacheCommand.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCLI.Commands
+{
+    using System;
+    using System.Diagnostics;
+    using System.IO;
+    using System.Threading.Tasks;
+    using CommandLine;
+    using Microsoft.WingetCreateCLI.Logging;
+    using Microsoft.WingetCreateCLI.Properties;
+    using Microsoft.WingetCreateCLI.Telemetry;
+    using Microsoft.WingetCreateCLI.Telemetry.Events;
+    using Microsoft.WingetCreateCore;
+
+    /// <summary>
+    /// Command to manage downloaded installers found in the %TEMP%/wingetcreate folder.
+    /// </summary>
+    [Verb("cache", HelpText = "CacheCommand_HelpText", ResourceType = typeof(Resources))]
+    public class CacheCommand : BaseCommand
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to delete all downloaded installers found in cached.
+        /// </summary>
+        [Option('c', "clean", SetName = nameof(Clean), Required = true, HelpText = "Clean_HelpText", ResourceType = typeof(Resources))]
```

Comment:
Why are these all Required:true?

Reply:
Similar to the token command, I have to make these all required or else someone could run 'wingetcreate cache' and no parsing errors would appear. Another alternative would be to check if these values are set and then show an error message, but I would want parsing errors to be handled by the commandline parser and not by the command

Reply:
It does result in some confusing output, but unfortunately CommandLine doesn't have a way to do mutually exclusive option sets :( I guess this is okay, if not ideal

Reply:
This issue tracks enabling this [https://github.com/commandlineparser/commandline/issues/419](https://github.com/commandlineparser/commandline/issues/419)